assemblyline/odm/base.py

```diff
@@ -1055,7 +1067,7 @@ def markdown(cls, toc_depth=1, defaults=None) -> Union[str, Dict]:
         markdown_content += f"{'#'*toc_depth} {cls.__name__}\n> {cls.__description}\n\n"
 
         # Table
-        table = "| Field | Type | Description | Required | Default |\n| :--- | :--- | :--- | :--- | :--- |\n"
+        table = '| Field | Type | Description | Required | Default |\n| :--- | :--- | :--- | :--- | :--- |\n'
```

Comment:
why

Reply:
Might've happened when I was trying to insert the div for spacing, will fix that